test-app-sitebricks.yaml

```diff
@@ -51,9 +51,9 @@ paths:
           description: "invalid id of user"
```

Comment:
so, let us be clear about that, you are expecting when a user is registering to provide he's own id?